integration-tests/api-gateway/session/session-multiple-names-test.ts

```diff
@@ -0,0 +1,27 @@
+import { createMultipleNamesSession } from "../endpoints";
+import { clearItemsFromTables } from "../../resources/dynamodb-helper";
+describe("Given the session is valid", () => {
+  let sessionId: string;
+
+  afterEach(async () => {
+    await clearItemsFromTables(
+      {
+        tableName: "person-identity-common-cri-api",
+        items: { sessionId: sessionId },
+      },
+      {
+        tableName: "session-common-cri-api",
+        items: { sessionId: sessionId },
+      }
+    );
+  });
+
+  it("Should receive a valid session id when /session endpoint is called", async () => {
+    const sessionResponse = await createMultipleNamesSession();
```

Comment:
This is interesting to me - what part of the session handler are we trying to test here? I didn't realise that it had extra logic to handle multiple names

Reply:
This was my bad, I was trying to get the logic working in session, but forgot to remove it. It's only added in the `check` folder